application.go

```diff
@@ -0,0 +1,134 @@
+package civogo
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+// Application is the struct for the Application model
+type Application struct {
+	Name        string        `json:"name" validate:"required" schema:"name"`
+	AccountID   string        `json:"account_id"`
```

Comment:
```suggestion

```
AccountID can be derived from API key.

Reply:
I need to use it in the CLI, when creating an application. For example, once a user creates an application, the cli's response will be - `fmt.Printf("Created an app %s, will be available at https://%s.%s.civo.app after pushing the code\n", utility.Green(app.Name), utility.Green(app.Name), utility.Green(app.AccountID))`

Do you have any other way of displaying it to the user?

Reply:
Maybe the default app domain should just be created like any other one and returned in the list of domains for the app when creating.

Reply:
Yes, it will be part of the API response in Domains field.